en/docs/references/ballerina-expression-syntax.md

```diff
@@ -37,6 +37,26 @@ To initialize a variable of type `float` in Choreo, you can use one of the follo
   +950.930
   ```
 
+### decimal
+
+The `decimal` type supports integer values with up to 34 decimal points and supports the same arithmetic operators as `int`. When entering literals, you can use `d` as the suffix.
+
+This type does not support `infinity`, `NaN`, or negative zero.
+
+The `decimal` type is a separate basic type and therefore counts as `anydata`.
+
+###### Examples:
+
+To initialize a variable of type `decimal` in Choreo, you can use one of the following expressions:
+
+  ```ballerina
+  0.33536363363636391745284629571635428
+  
+  -8593.9926666666666666663333333333
+  
+  +950.930999999992727272777777777774
```

Comment:
Shall we add an example with the `d` suffix also?

Reply:
Done